adding plan attribute to fix #159 (#160)
terraform-providers/terraform-provider-cloudflare: website/docs/r/zone.html.markdown

* `zone` - (Required) The DNS zone name which will be added.
* `paused` - (Optional) Boolean of whether this zone is paused (traffic bypasses Cloudflare). Default: false.
* `jump_start` - (Optional) Boolean of whether to scan for DNS records on creation. Ignored after zone is created. Default: false.

> * `plan` - (Optional) The name of the commercial plan to apply to the zone, can be updated once the one is created; one of `Free Website`, `TODO`, `TODO`, `TODO` 

Reviewer: ```suggestion
* `plan` - (Optional) The name of the commercial plan to apply to the zone. Valid values: `free`, `business`, `pro`, `enterprise`.
```

Author: As discussed, used literal names from Cloudflare-go
